charts/openforms/values.yaml

```diff
@@ -259,6 +273,9 @@ existingSecret: null
 existingConfigurationSecrets: null
 # If no secret already exists with the values needed for django-setup-configuration, create it with the following name
 configurationSecretsName: ""
+# Reference to existing tls secret
+existingTlsSecret: null
+tlsSecretName: ""
```

Comment:
Maybe good to document that if no `existingTlsSecret` is specified, then it will be created with the name specified by `tlsSecretName` (which defaults to openforms-tls ... depending on fullname :smile: )